### ISSUE TITLE
Could not resolve com.facebook.react:react-native:0.19.+.

### ISSUE THREAD

**Issue by Msiavashi** [facebook/react-native#5884]
im having this problem after upgrading from 0.17 to 0.19 when i run "react-native run-android" command:

```
Starting JS server...
Starting the packager in a new window is not supported on Windows yet.
Please start it manually using 'react-native start'.
We believe the best Windows support will come from a community of people
using React Native on Windows on a daily basis.
Would you be up for sending a pull request?
Building and installing the app on the device (cd android && gradlew.bat install
Debug)...

FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring project ':app'.
> Could not resolve all dependencies for configuration ':app:_debugCompile'.
   > Could not resolve com.facebook.react:react-native:0.19.+.
     Required by:
         sess:app:unspecified
      > Could not resolve com.facebook.react:react-native:0.19.+.
         > Failed to list versions for com.facebook.react:react-native.
            > Unable to load Maven meta-data from https://jcenter.bintray.com/co
m/facebook/react/react-native/maven-metadata.xml.
               > Could not GET 'https://jcenter.bintray.com/com/facebook/react/r
eact-native/maven-metadata.xml'.
                  > Connection to https://jcenter.bintray.com refused

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug
option to get more log output.

BUILD FAILED

Total time: 49.111 secs
Could not install the app on the device, read the error above for details.
Make sure you have an Android emulator running or a device connected and have
set up your Android development environment:
https://facebook.github.io/react-native/docs/android-setup.html
```

whats wrong ?

**Comment by bestander** [comment]
I've had a similar problem when Gradle did not accept SSL certificate from external website.
 > Connection to https://jcenter.bintray.com refused

This is not related to React Native I reckon.
In version 0.21 we won't be distributing the jar via jcenter and it will be distributed with the npm package.

**Comment by Msiavashi** [comment]
well , how can i download the v0.21 i've also tried downgrading back to 0.17 but the problem is still there .

**Comment by bestander** [comment]
Probably you could download the jar manually and place it where gradle is
supposed to put it.

21-rc is coming next week

**Comment by Heaven1619** [comment]
I installed 21-rc build but the problem still persist.

**Comment by bestander** [comment]
@Heaven1619 could you post a log error

**Comment by Heaven1619** [comment]
$ react-native -v
react-native-cli: 0.1.10
react-native: 0.21.0-rc

$ react-native run-android
Starting JS server...
Building and installing the app on the device (cd android && ./gradlew installDebug)...

FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring project ':app'.
> Could not resolve all dependencies for configuration ':app:_debugCompile'.
   > Could not resolve com.facebook.react:react-native:0.19.+.
     Required by:
         Mad:app:unspecified
      > Could not resolve com.facebook.react:react-native:0.19.+.
         > Failed to list versions for com.facebook.react:react-native.
            > Unable to load Maven meta-data from https://jcenter.bintray.com/com/facebook/react/react-native/maven-metadata.xml.
               > Could not GET 'https://jcenter.bintray.com/com/facebook/react/react-native/maven-metadata.xml'.
                  > Connection to https://jcenter.bintray.com refused

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED

Total time: 4 mins 15.891 secs
Could not install the app on the device, read the error above for details.
Make sure you have an Android emulator running or a device connected and have
set up your Android development environment:
https://facebook.github.io/react-native/docs/android-setup.html

**Comment by Msiavashi** [comment]
See if my answer helps : http://stackoverflow.com/questions/35394570/react-native-build-failed-with-an-exception

**Comment by bestander** [comment]
@Msiavashi thanks for your help.

@Heaven1619 0.21 has just been cut, I think we are not linking to the jar properly and it still downloads the react-native.jar v 0.20.1 from Maven.
We will fix this in the next 2 weeks when 0.21 will become stable.

**Comment by ThaJay** [comment]
https://jcenter.bintray.com/com/facebook/react/react-native/maven-metadata.xml still only goes up to 0.20.1 but run-android is still using it. How can I get the java version of 0.21.0 of better?
